metering/Containerfile

```diff
@@ -0,0 +1,38 @@
+ARG IMAGE_VERSION=latest
+FROM python:${IMAGE_VERSION}
```

Comment:
Use an arg PYTHON_VERSION here. Which defaults to the Python version that should be used.

Reply:
using now python 3.10